com.unity.render-pipelines.high-definition/Documentation~/Override-Volumetric-Clouds.md

```diff
@@ -118,6 +118,7 @@ When importing these two map textures, disable **sRGB**. For best results, do no
 | **Property**                     | **Description**                                              |
 | -------------------------------- | ------------------------------------------------------------ |
 | **Temporal Accumulation Factor** | The amount of temporal accumulation to apply to the clouds. Temporal accumulation increases the visual quality of clouds by decreasing the noise. A higher value produces better quality clouds, but can create [ghosting](Glossary.md#ghosting). |
+| **High Speed Ghosting Reductionr** | Indicates whether to remove the ghosting induced by the temporal accumulation. When active, it may introduce some flickering at lower temporal accumulation factors. |
```

Comment:
To reduce the width of the label, I'd just suggest 

- Anti-ghosting
- High Speed Anti-ghosting

I would also suggest we change the second sentence into:

> When active, a higher number of primary steps might be required to prevent potential flickering.

Because flickering seems to be visible at any level of accumulation factors (but of course a lot more with lower accumulation).

Reply:
just realized there is an extra R in the name of the property :(